tools/wptrunner/wptrunner/browsers/edge.py

```diff
@@ -82,6 +84,13 @@ def start(self, **kwargs):
 
     def stop(self, force=False):
         self.server.stop(force=force)
+        time.sleep(3)
+        # close Edge process if it is still running
+        edge_proc_name = 'MicrosoftEdge.exe'
+        procs = subprocess.check_output(['tasklist', '/fi', 'ImageName eq ' + edge_proc_name])
+        if edge_proc_name in procs:
+            subprocess.call(['taskkill.exe', '/f', '/im', 'microsoftedge*']) # close all Edge browser instances
+        self.logger.info("stopped")
```

Comment:
Did you mean to include this or was it for debugging? Can you move the logging inside the if branch and make it a bit more informative?

Reply:
This was used for debugging, removed it.